toolkit/Makefile

```diff
@@ -142,6 +143,25 @@ SRPM_URL_LIST      ?= https://packages.microsoft.com/azurelinux/$(RELEASE_MAJOR_
 include $(SCRIPTS_DIR)/utils.mk
 
 ######## REMAINING BUILD FLAGS ########
+# DAILY_BUILD_ID and DAILY_BUILD REPO while similar are mutually exclusive.
+# 
+# DAILY_BUILD_ID is points the build tools to the latest daily built 
+# packages as part of a tight local workflow.  The DAILY_BUILD_ID argument 
+# takes the form: V-v-YYYYMMDD where V-v is the Major-Minor branch
+#
+# DAILY_BUILD_REPO also points to the latest daily built packages.  However 
+# this is intended for internal teams to access the packages at build time and
+# runtime.  The argument takes the form of a repo file supplied by the
+# Azure Linux Team's daily artifact feed.  The repo file may be provisioned
+# in a downstream image so that it has access to the repo at runtime.  The
+# DAILY_BUILD_REPO argument takes a path to the daily-mariner.repo file.
+# 
+# their pipelines on a daily basis. The repo file may be provisioned to their image \ 
```

Comment:
The trailing newline here breaks the `\`, but if that gets cleaned up (ie by an auto clean on save) it will break the next line.
```suggestion
# their pipelines on a daily basis. The repo file may be provisioned to their image
```